decidim-meetings/spec/system/explore_meetings_spec.rb

```diff
@@ -27,6 +27,44 @@
       end
     end
 
+    context "when checking withdrawn meetings" do
+      context "when there are no withrawn meetings" do
+        let!(:meeting) { create_list(:meeting, 3, :published, component: component) }
+
+        before do
+          visit_component
+          click_link "See all withdrawn meetings"
+        end
+
+        # it_behaves_like "accessible page"
+
+        it "shows an empty page with a message" do
+          expect(page).to have_content("No meetings match your search criteria or there isn't any meeting scheduled.")
+          within ".callout.warning", match: :first do
+            expect(page).to have_content("You are viewing the list of meetings withdrawn by their authors.")
+          end
+        end
+      end
+
+      context "when there are withrawn meetings" do
+        let!(:withdrawn_meetings) { create_list(:meeting, 3, :withdrawn, :published, component: component) }
+
+        before do
+          visit_component
+          click_link "See all withdrawn meetings"
+        end
+
+        # it_behaves_like "accessible page"
```

Comment:
Any reason why this is commented out?

Reply:
This is commented out as those tests have been extracted from our test, and Decidim is not accessible ready on that page, and fixing it is not in scope of this particular PR.